### DISCUSSION TITLE
Ideal way to Test Public and Individual Tenants?

### DISCUSSION THREAD

**Discussion by allidoisace** [django-tenants/django-tenants#848] [Q&A]
Do I just need to create a public tenant and a normal tenant in my unit test?

**Comment by matt256** [comment]
Django-tenant can create a test tenant for you. This person got it figured out. Check out their answer.
https://github.com/django-tenants/django-tenants/issues/813
You have to specify your schema context to be either your public or tenant schema. that's the trick
